app/models/event.rb

```diff
@@ -0,0 +1,8 @@
+class Event < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :attendees
+  has_many :users, through: :attendees
+
+  scope :past, -> { where('datetime <= ?', Time.zone.now) }
```

Comment:
For `past` events, it makes more sense to use `datetime < ?` instead of `datetime <= ?` to avoid chances of including the current event in the `past` events.